app/src/main/java/net/skyscanner/backpack/demo/stories/Calendar2Story.kt

```diff
@@ -57,9 +58,8 @@ class Calendar2Story : Story() {
 
   private val calendar by unsafeLazy { view!!.findViewById<BpkCalendar>(R.id.calendar2)!! }
   private val type by unsafeLazy { arguments!!.getSerializable(TYPE) as Type }
-  private val now = LocalDate.now()
+  private val now = LocalDate.of(2019, 1, 1)
```

Comment:
Since we use automatic screenshot taking, I decided to make screenshots more consistent and remove inconsistent elements (such as today date and random in price generator).